.circleci/config.yml

```diff
@@ -77,4 +82,4 @@ jobs:
       - store_test_results:
           path: test-results
       - store_artifacts:
-          path: htmlcov.tar
\ No newline at end of file
+          path: htmlcov.tar
```

Comment:
Isnt this the same line?

Reply:
I think the newline got deleted, which is why this shows a diff